lib/apiservers/engine/backends/container.go

```diff
@@ -858,6 +856,30 @@ func createNewAttachClientWithTimeouts(connectTimeout, responseTimeout, response
 	return plClient, transport
 }
 
+func IsContainerRunning(containerID string) (bool, error) {
```

Comment:
Doesn't look like you need the bool.  Just return an error and check if err != nil.